connectors/byoei.py

```diff
@@ -94,6 +99,9 @@ def __init__(
         self.chunk_mem_size = chunk_mem_size * 1024 * 1024
         self.max_concurrent_bulks = max_concurrency
         self.bulk_tasks = ConcurrentTasks(max_concurrency=max_concurrency)
+        self.indexed_document_count = 0
+        self.indexed_document_volume = 0
+        self.deleted_document_count = 0
```

Comment:
It will be used in PR https://github.com/elastic/connectors-python/pull/513:
https://github.com/elastic/connectors-python/blob/60722d4d250c1e0dd58f35e8871de7b0f5d2b9cc/connectors/byoei.py#L541-L545